src/mscorlib/src/System/Collections/Generic/List.cs

```diff
@@ -41,7 +41,7 @@ public class List<T> : IList<T>, System.Collections.IList, IReadOnlyList<T>
         [NonSerialized]
         private Object _syncRoot;
         
-        static readonly T[]  _emptyArray = new T[0];        
+        static readonly T[]  _emptyArray = new T[0];
```

Comment:
The whitespace-only changes to this file aren't necessary; the whole file can just be reverted.